src/inspector/view/qml/MuseScore/Inspector/notation/ambituses/AmbitusSettings.qml

```diff
@@ -42,41 +42,41 @@ Column {
     spacing: 12
 
     readonly property var tpcListModel: [
-        { text: "C♭♭", value: AmbitusTypes.TPC_C_BB },
-        { text: "C♭", value: AmbitusTypes.TPC_C_B },
-        { text: "C", value: AmbitusTypes.TPC_C },
-        { text: "C♯", value: AmbitusTypes.TPC_C_S },
-        { text: "C♯♯", value: AmbitusTypes.TPC_C_SS },
-        { text: "D♭♭", value: AmbitusTypes.TPC_D_BB },
-        { text: "D♭", value: AmbitusTypes.TPC_D_B },
-        { text: "D", value: AmbitusTypes.TPC_D },
-        { text: "D♯", value: AmbitusTypes.TPC_D_S },
-        { text: "D♯♯", value: AmbitusTypes.TPC_D_SS },
-        { text: "E♭♭", value: AmbitusTypes.TPC_E_BB },
-        { text: "E♭", value: AmbitusTypes.TPC_E_B },
-        { text: "E", value: AmbitusTypes.TPC_E },
-        { text: "E♯", value: AmbitusTypes.TPC_E_S },
-        { text: "E♯♯", value: AmbitusTypes.TPC_E_SS },
-        { text: "F♭♭", value: AmbitusTypes.TPC_F_BB },
-        { text: "F♭", value: AmbitusTypes.TPC_F_B },
-        { text: "F", value: AmbitusTypes.TPC_F },
-        { text: "F♯", value: AmbitusTypes.TPC_F_S },
-        { text: "F♯♯", value: AmbitusTypes.TPC_F_SS },
-        { text: "G♭♭", value: AmbitusTypes.TPC_G_BB },
-        { text: "G♭", value: AmbitusTypes.TPC_G_B },
-        { text: "G", value: AmbitusTypes.TPC_G },
-        { text: "G♯", value: AmbitusTypes.TPC_G_S },
-        { text: "G♯♯", value: AmbitusTypes.TPC_G_SS },
-        { text: "A♭♭", value: AmbitusTypes.TPC_A_BB },
-        { text: "A♭", value: AmbitusTypes.TPC_A_B },
-        { text: "A", value: AmbitusTypes.TPC_A },
-        { text: "A♯", value: AmbitusTypes.TPC_A_S },
-        { text: "A♯♯", value: AmbitusTypes.TPC_A_SS },
-        { text: "B♭♭", value: AmbitusTypes.TPC_B_BB },
-        { text: "B♭", value: AmbitusTypes.TPC_B_B },
-        { text: "B", value: AmbitusTypes.TPC_B },
-        { text: "B♯", value: AmbitusTypes.TPC_B_S },
-        { text: "B♯♯", value: AmbitusTypes.TPC_B_SS }
+        { text: qsTrc("global", "C♭♭"), value: AmbitusTypes.TPC_C_BB },
```

Comment:
Using 'global' as the context here rather than 'inspector', as a good part of these strigs (C, C♯, D♭, D, D♯, E♭, E, F, F♯, G, G♯, A, A♯, B♭, and B) are available in the global context already.